src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -89,14 +89,25 @@ public void start(Map<String, String> props) {
         remainingRetries = config.maxRetries;
     }
 
+    private void attemptPutItem(SinkRecord record) {
+        try {
+            client.putItem(tableName(record), toPutRequest(record).getItem());
+        } catch (AmazonDynamoDBException e) {
```

Comment:
Would there happen to be another exception that could be thrown here from aws that we'd like to catch?

Reply:
this is our project now so we can add more as they come up

Reply:
![53697461](https://user-images.githubusercontent.com/35939863/38948149-88a3a090-42f3-11e8-89f8-df7655bbc96b.jpg)